packages/@lwc/engine/src/framework/upgradable-element.ts

```diff
@@ -48,11 +49,11 @@ function defineUpgradableElement(tagName: string): boolean {
     return true;
 }
 
-export function registerTagName(tagName: string) {
+export function registerTagName(tagName: string): boolean {
```

Comment:
I don't think we should return the result of this operation, it is not wise. This is supposed to be called multiple times for the same tagName... returning false means nothing, it probably was defined earlier.